packages/showcase/src/app/components/pill/metadata.ts

```diff
@@ -0,0 +1,182 @@
+import { ComponentMetadata } from "../../../common/types";
+import { chameleonImportType } from "../../../services/template-language/create-template";
+import {
+  pillStatusModel,
+  devStatusModel,
+  priorityLevelModel,
+  taskCategoryModel
+} from "./models";
+import { CSS } from "../../../common/html";
+
+const IMPORTS = [chameleonImportType("ComboBoxModel")];
+
+const STATES = [
+  {
+    name: "pillModel",
+    type: "ComboBoxModel",
+    value: pillStatusModel
+  }
+];
+
+const TAG = CSS.PILL.TAG;
+const SELECTOR = CSS.PILL.SELECTOR;
+
+const PROPERTIES = [
+  { name: "model", value: "pillModel", state: true },
+  { name: "accessibleName", value: "status" },
+  { name: "placeholder", value: "Select a status" }
+];
+
+export const pillMetadata = {
+  title: "Pill",
+  description:
+    "A pill in web development is a compact, rounded UI element used to display labels, categories, or selections. It visually stands out with a subtle, often colorful background, and is commonly used to represent tags, filters, or options. Pills can be interactive, allowing users to select or remove them, enhancing usability and clarity in filtering or selection tasks.",
+
+  codeSnippets: {
+    placeholder: {
```

Comment:
We should add the cases with/without label (same as the combo-box):
https://mercury-showcase.genexus.com/mercury/components/combo-box